akka-actor/src/main/scala/akka/actor/LightArrayRevolverScheduler.scala

```diff
@@ -59,10 +59,12 @@ class LightArrayRevolverScheduler(
     val dn = d.toNanos
     if (dn <= 0)
       Duration.Zero
-    else if (dn > Long.MaxValue - (tickNanos - 1))
+    else if (dn > Long.MaxValue - tickNanos)
```

Comment:
i thought it prevents overflow in `checkQueue` method
`(ticks * tickNanos) + tickNanos - 1`, but i was wrong, it starts with `time - start`
do i change this code to prevent overflow?